lua/obsidian/commands/quick_switch.lua

```diff
@@ -12,11 +12,28 @@ return function(client, _)
       if not has_telescope then
         return false
       end
+
       -- Search with telescope.nvim
+      local custom_actions = {
+        -- https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/actions/init.lua
+        obsidian_new = function(prompt_bufnr)
+          local query = require("telescope.actions.state").get_current_line()
+          require("telescope.actions").close(prompt_bufnr)
+          local new = require "obsidian.commands.new"
+          new(client, { args = query })
```

Comment:
This gave me the idea to make a better API for calling our commands directly. Once https://github.com/epwalsh/obsidian.nvim/pull/298/files merges we can change this to:

```suggestion
          client:command("ObsidianNew", { args = query })
```

Reply:
Nice little improvement. I'll rebase and integrate it.

Reply:
done